src/marqo/core/embed/embed.py

```diff
@@ -85,14 +85,15 @@ def embed_content(
         # Decide on the prefix 
 
         # For backwards compatibility
-        if marqo_index.model.text_query_prefix is None or marqo_index.model.text_chunk_prefix is None:
-            prefix = ""
-        elif content_type == EmbedContentType.Query:
+        
+        if content_type == EmbedContentType.Query:
             prefix = marqo_index.model.get_text_query_prefix()
         elif content_type == EmbedContentType.Document:
             prefix = marqo_index.model.get_text_chunk_prefix()
         elif content_type is None:
```

Comment:
Is None a valid value here?

Reply:
Yes, None is expected according to Joshua's discussion with Owen